### PR TITLE
Adds Retrieve Domain Restricted Groups GPOs Procedure

### PR DESCRIPTION
**Pull request by w0rk3r**
Get all GPOs that modify local memberships through Restrict Groups

### CODE REVIEW

File: discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
+id: 7de00454-d90c-4b72-97b1-ee1bb37f31e7
+name: Retrive Domain Restrict Groups GPOs 
+description: Get all GPOs that modify local memberships through Restrict Groups
```
Comment (khyberspache) [inline discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml:3]: restricted

**Review by khyberspache — CHANGES_REQUESTED** [review]

File: discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml (hunk 1 above)
Comment (w0rk3r) [reply discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml]: Solved

File: discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+id: 7de00454-d90c-4b72-97b1-ee1bb37f31e7
+name: Retrive Domain Restrict Groups GPOs
```
Comment (khyberspache) [inline discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml:2]: restricted here as well

Comment (khyberspache) [reply discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml:2]: Retrieve

Comment (w0rk3r) [reply discovery/7de00454-d90c-4b72-97b1-ee1bb37f31e7.yml:2]: Oh

**Review by khyberspache — APPROVED** [review]

**Pull request merged**